### PR TITLE
Vstest args for dotnet cli

### PR DESCRIPTION
**Pull request by mayankbansal018**
- Allow dotnet CLI to take multiple inputs for logger, & TAP
- Complete PR: https://github.com/Microsoft/vstest/pull/1574 , then update dotnet sdk here

Fixes: https://github.com/Microsoft/vstest/issues/1319

### CODE REVIEW

File: src/dotnet/commands/dotnet-test/TestCommandParser.cs [hunk 1]
```diff
@@ -40,19 +40,19 @@ internal static class TestCommandParser
                   Create.Option(
                         "-a|--test-adapter-path",
                         LocalizableStrings.CmdTestAdapterPathDescription,
-                        Accept.ExactlyOneArgument()
+                        Accept.OneOrMoreArguments()
                               .With(name: LocalizableStrings.CmdTestAdapterPath)
-                              .ForwardAsSingle(o => $"-property:VSTestTestAdapterPath={o.Arguments.Single()}")),
+                              .ForwardAsSingle(o => $"/p:VSTestTestAdapterPath=\"{string.Join(";", o.Arguments)}\"")),
```
Comment (smadala) [inline src/dotnet/commands/dotnet-test/TestCommandParser.cs]: Any reasons changing from `-property` to `/p`?

File: src/dotnet/commands/dotnet-test/Program.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public static TestCommand FromArgs(string[] args, string msbuildPath = null)
             var msbuildArgs = new List<string>()
             {
                 "-target:VSTest",
-                "-verbosity:quiet",
+                "-verbosity:minimal",
```
Comment (smadala) [inline src/dotnet/commands/dotnet-test/Program.cs]: Check what is impact of [PR](https://github.com/dotnet/cli/pull/9267/files#diff-ab184f453b5f0090121f80776937e88bR63) on this change.

File: src/dotnet/commands/dotnet-test/TestCommandParser.cs (hunk 1 above)
Comment (peterhuene) [reply src/dotnet/commands/dotnet-test/TestCommandParser.cs]: The CLI source base actually standardized on `-property`, so let's keep it as it was.

File: src/dotnet/commands/dotnet-test/TestCommandParser.cs [hunk 3]
```diff
@@ -40,19 +40,19 @@ internal static class TestCommandParser
...
                         LocalizableStrings.CmdLoggerDescription,
-                        Accept.ExactlyOneArgument()
+                        Accept.OneOrMoreArguments()
                               .With(name: LocalizableStrings.CmdLoggerOption)
                               .ForwardAsSingle(o =>
                                     {
                                           var loggersString = string.Join(";", GetSemiColonEscapedArgs(o.Arguments));
 
-                                          return $"-property:VSTestLogger={loggersString}";
+                                        return $"/p:VSTestLogger=\"{loggersString}\"";
```
Comment (peterhuene) [inline src/dotnet/commands/dotnet-test/TestCommandParser.cs]: Same as above.

**Comment by peterhuene** [comment]
Also, could we rebase this off of `release/2.1.4xx` rather than having the merge commit (I wouldn't mind a squash too, but we can do that at merge time)? 

Note that `release/2.1.4xx` now has the fix for the verbosity issue.

**Comment by peterhuene** [comment]
@dotnet-bot test this please.

File: src/dotnet/commands/dotnet-test/TestCommandParser.cs (hunk 1 above)
Comment (mayankbansal018) [reply src/dotnet/commands/dotnet-test/TestCommandParser.cs]: Addressed the comment

**Comment by mayankbansal018** [comment]
@peterhuene , I've rebased the branch from 2.1.4xx, please do a squash merge if the changes look good to you.

**Comment by mayankbansal018** [comment]
@livarcocc , @peterhuene , I'm not sure why the wip check is not passing, but can you please take a look at this PR, & merge it.

**Comment by peterhuene** [comment]
It's not passing because one of your commits has `WIP` in it.

**Pull request closed without merging**